internal/cache/cache.go

```diff
@@ -27,21 +27,23 @@ const dirMode = 0700
 const fileMode = 0644
 
 func readVersion(dir string) (v uint, err error) {
+	defer func() {
+		if err != nil {
+			err = errors.Wrap(err, "readVersion")
```

Comment:
Do we still get an accurate stacktrace which shows which return was used? `errors.Wrap` already ignores `nil` errors.

Reply:
It still shows the line of the return at line 40, but it also adds the anonymous function to the trace.